scripts/functions/clean_orphan_elements/clean_orphan_networks.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from scripts import constant
+
+def get_network_infos_from_study(study):
+    return study["rootNetworkInfos"]
+
+# turn studies_response_json into a network uuids list
+def get_network_uuids_from_studies(studies):
+    root_network_infos = map(get_network_infos_from_study, studies)
+    root_network_infos_list = [element for sub_list in list(root_network_infos) for element in sub_list]
+    network_uuids = []
+    for root_network_info in root_network_infos_list:
+        network_uuids.append(root_network_info['networkInfos']["networkUuid"])
+    return network_uuids
+
+
+def get_network_uuid_from_network(network):
+    return network["uuid"]
+
+def delete_networks(network_uuids, dry_run):
+    if dry_run:
+        for orphan_n in network_uuids:
+            print("DELETE " + constant.DELETE_NETWORKS + "/" + orphan_n)
+    else:
+        for orphan_n in network_uuids:
+            requests.delete(constant.DELETE_NETWORKS + "/" + orphan_n)
+
+# DELETING ORPHAN NETWORKS IN NETWORK STORE SERVER
+def delete_orphan_networks(dry_run):
+    print("/// Orphan networks deletion ///")
+    # GET EXISTING NETWORKS AMONG EXISTING STUDIES
+    print("Getting used networks from existing studies: " + constant.GET_SUPERVISION_STUDIES)
+    studies_response = requests.get(constant.GET_SUPERVISION_STUDIES)
+    studies_response_json = studies_response.json()
+    network_uuids_used_in_studies = get_network_uuids_from_studies(studies_response_json)
+    print("Done")
+
+    # GET NETWORKS SAVED IN NETWORK STORE SERVER
+    print("Getting networks from network store server : " + constant.GET_NETWORKS)
+    get_networks_response = requests.get(constant.GET_NETWORKS)
+    get_networks_response_json = get_networks_response.json()
+    all_networks_uuids = list(map(get_network_uuid, get_networks_response_json))
+    print("Done")
+
+    # GET ORPHANS NETWORKS - NETWORKS IN NETWORK STORE SERVER WHICH ARE NOT KNOWN IN STUDY SERVER
+    print("Computing orphan networks")
+    orphan_networks = []
+    for network_uuid in all_networks_uuids:
+        if network_uuid not in network_uuids_used_in_studies:
+            orphan_networks.append(network_uuid)
```

Comment:
/// Orphan networks deletion ///
Getting used networks from existing studies: http://172.17.0.1:5001/v1/supervision/studies
Study referenced networks UUIDs: 34
Done
Getting networks from network store server : http://172.17.0.1:8080/v1/networks
Store networks UUIDs: 67
Done
Computing orphan networks
Done
Deleting the following 33 orphan networks : 

See the log above, it seems network-store persistes also root-network so we have 33 orphan in my db..